packages/ui-react/src/components/nft/components/nft-media.tsx

```diff
@@ -0,0 +1,21 @@
+import { useNFT } from "./provider";
+
+export function NFTMedia() {
+	const { nftData } = useNFT();
+
+	const imageUrl = nftData?.image_url;
```

Comment:
@iyansr do you hv example data for blockscout nft url?
many NFT provide ipfs hash so we need explicit ipfs gateway control on the image to be shown 

(reference `getIpfsGatewayUrl`)